ui/client/actions/nk/notifications.ts

```diff
@@ -0,0 +1,33 @@
+import {ThunkAction} from "../reduxTypes";
+
+export type NotificationActions =
+  | UpdateBackendNotifications
+  | MarkBackendNotificationRead
+
+export type UpdateBackendNotifications = {
+    type: "UPDATE_BACKEND_NOTIFICATIONS",
+    notificationIds: string[]
+}
+
+export function updateBackendNotifications(notificationIds: string[]): ThunkAction {
```

Comment:
no side effects so simple action creator without dispatch is enough here

Reply:
+1 true...